<launch>

  <!-- these are the arguments you can pass this launch file, for example paused:=true -->
  <arg name="load_modelfile_path" default="true"/>

  <rosparam file="$(find skinsim_model_generator)/config/model_params.yaml" command="load" />
 
  <!-- set load_jointfile_path flag -->
  <group if="$(arg load_modelfile_path)">
    <param name="sdf_filename"  value="$(find skinsim_model_generator)/generated_models/model.sdf" />
    <param name="joint_config_filename"  value="$(find skinsim_model_generator)/generated_models/joint_names.yaml" />
  </group>
 
  <node name="skinsim_model_generator" pkg="skinsim_model_generator" type="skinmodel_sdf_generator" respawn="false" />
  
</launch>
